src/odinapi/views/views.py

```diff
@@ -38,7 +38,8 @@
 
 # Make linter happy
 use_agg
-
+SWAGGER.add_parameter("freqmode", "path", int)
+SWAGGER.add_parameter("scanno", "path", int)
 SWAGGER.add_parameter('date', 'path', str, string_format='date')
 SWAGGER.add_type('freqmode_info', {
     'Backend': str,
```

Comment:
Have you checked how the swagger docs renders with your changes?

Reply:
No, but there are a lot of add_parameter calls scattered around the code  so i don't think order matters. Probably we should just cut the swagger-try-to-integrate-the-code and replace with a standard openapi yaml file at some point...